src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpTraceExporter.cs

```diff
@@ -75,15 +75,25 @@ internal OtlpTraceExporter(OtlpExporterOptions options, OtlpCollector.TraceServi
             }
             else
             {
-                if (options.Endpoint.Scheme == Uri.UriSchemeHttps)
+                if (options.Endpoint.Scheme != Uri.UriSchemeHttp && options.Endpoint.Scheme != Uri.UriSchemeHttps)
```

Comment:
This is very defensive: it seems that the main concern is with Unix Domain Sockets but typically those are not wrapped around TCP and actually use file security instead. That said I don't see a problem with blocking its support until the next version.

Reply:
If you think it's safe to remove this, I'm not opposed to it. I just haven't tried out unix sockets yet. The docs for how to do this with the netstandard2.1 library gave me some pause https://docs.microsoft.com/en-us/aspnet/core/grpc/interprocess?view=aspnetcore-5.0#client-configuration. I'm not sure we have a way to support this yet.

Reply:
Sorry, missed this earlier @alanwest. Let's keep it as it is so stays aligned with what was tested.